src/lib/expansion/expansion-panel-header.ts

```diff
@@ -62,8 +67,22 @@ import {MdExpansionPanel, EXPANSION_PANEL_ANIMATION_TIMING} from './expansion-pa
     ]),
   ],
 })
-export class MdExpansionPanelHeader {
-  constructor(@Host() public panel: MdExpansionPanel) {}
+export class MdExpansionPanelHeader implements OnDestroy {
+  private _parentChangeSubscription: Subscription | null = null;
+
+  constructor(
+    @Host() public panel: MdExpansionPanel,
+    private _changeDetectorRef: ChangeDetectorRef) {
+
+    // Since the toggle state depends on an @Input on the panel, we
+    // need to  subscribe and trigger change detection manually.
+    this._parentChangeSubscription = merge(
+      panel.opened,
+      panel.closed,
+      filter.call(panel._inputChanges, changes => !!changes.hideToggle)
+    )
+    .subscribe(() => this._changeDetectorRef.markForCheck());
```

Comment:
I can give it a try, but I feel like the manual change detection might cause it not to fail as expected.